### PR TITLE
docs(components): Adding InlineLabel to New Docs Site [JOB-110397]

### PR DESCRIPTION
**Pull request by Beccaneer**
### Changed

Added the InlineLabel component to the new docs site.

## Testing

Run the docs site from the packages/site directory with `npm run dev`.
You should see an 'InlineLabel' option after clicking on Components.

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by scotttjob — APPROVED** [review]
LGTM! Let me know if you need me to come back for another approval once you fix the title for the PR issue.

We should probably add some new entries to our setup, but I don't think it likes 'docs' as that part of the title

**Review by chris-at-jobber — APPROVED** [review]

**Pull request merged**
